wcfsetup/install/files/lib/util/StringUtil.class.php

```diff
@@ -841,6 +802,18 @@ public static function normalizeCsv($string): string
         return \implode(',', ArrayUtil::trim(\explode(',', $string)));
     }
 
+    private static function getNumberFormatter(): \NumberFormatter
+    {
+        static $formatters = [];
+
+        $locale = WCF::getLanguage()->getLocale();
+        if (!isset($formatters[$locale])) {
+            $formatters[$locale] = new \NumberFormatter($locale, \NumberFormatter::TYPE_DEFAULT);
```

Comment:
```suggestion
            $formatters[$locale] = new \NumberFormatter($locale, \NumberFormatter::DEFAULT_STYLE);
```

The type constants are used as the second parameter of `->format()`. `TYPE_DEFAULT == PATTERN_DECIMAL` which breaks the output format, because the default pattern is `null`.